slimCat/Utilities/StaticFunctions.cs

```diff
@@ -451,5 +451,19 @@ public static void TryOpenRightClickMenuCommand<T>(object sender, int ancestorLe
             { typeof(CharacterUpdateModel), x => ((CharacterUpdateModel)x.DataContext).TargetCharacter.Name }
         };
 
+        public static double GetScrollDistance(int scrollTicks, int fontSize)
+        {
+            var linesPerTick = SystemParameters.WheelScrollLines;
+            // This solves Windows 7 wheel setting "One screen at a time"
+            if (linesPerTick < 1)
+                linesPerTick = 10;
+
+            // An estimation for the height of each line
+            var lineSize = (fontSize * 0.89d) + 9.72d;
```

Comment:
I did a bit of trial and error with those - the basic idea was to minimize the variation that happens with different font sizes, and make the default font size 13 look "perfect" for scrolling.
I'm not sure what a better way might be, other than to make a dictionary and give each font size its own exact line size. It'd take some time but it'd be doable.
But I figure that's close enough? If you try it out, I figure you might like it.